src/interaction/interactiveTarget.js

```diff
@@ -2,6 +2,7 @@ var core = require('../core');
 
 
 core.DisplayObject.prototype.interactive = false;
+core.DisplayObject.prototype.dblClickDelay = 700;
```

Comment:
This seems like a really long time, basically if I am listening for clicks and doubleclicks the regular clicks are *significantly* delayed. 700ms is noticeable by the end user.